src/main/java/seedu/duke/command/AddEventCommand.java

```diff
@@ -40,6 +40,7 @@ public static String getCommandUsage() {
     }
 
     private static final String COMMAND_SUCCESSFUL_MESSAGE = "Added the following!";
+    // Command not expected to fail at this stage
```

Comment:
Shld this comment be above? or somewhere in execute? Looks a little out of place

Reply:
I placed it here because following the rest I thought we would expect a COMMAND_UNSUCCESSFUL_MESSAGE. Can remove comment as well.

Reply:
I've re-worded the comment to be clearer as to what it means so it doesn't look as weird